app/src/main/java/com/kamron/pogoiv/GoIVSettings.java

```diff
@@ -56,6 +62,22 @@ public void setPlayerTeam(int value) {
         editor.commit();
     }
 
+    public String getClipboardPreference() {
+        Log.d("NahojjjenClippy", "String representation of token train from settings: "
+                + prefs.getString(GOIV_CLIPBOARDSETTINGS, "error"));
+        return prefs.getString(GOIV_CLIPBOARDSETTINGS, "");
+    }
+
+    public void setClipboardPreference(ArrayList<ClipboardToken> tokens) {
+        SharedPreferences.Editor editor = prefs.edit();
+        String saveString = "";
+        for (ClipboardToken token : tokens) {
+            saveString += token.getStringRepresentation();
+        }
+        editor.putString(GoIVSettings.GOIV_CLIPBOARDSETTINGS, saveString);
+        editor.commit();
+    }
```

Comment:
To check: right now we don't explicitly commit other settings, check if we can be consistent here. Also, `commit` is discouraged.